src/main/java/org/ladocuploader/app/preparers/NoPermanentAddressMailingAddressPreparer.java

```diff
@@ -20,16 +20,20 @@ public Map<String, SubmissionField> prepareSubmissionFields(Submission submissio
     Map<String, Object> inputData = submission.getInputData();
 
     boolean noHomeAddress = inputData.getOrDefault("noHomeAddress[]", List.of("false")).equals(List.of("true"));
+    boolean noMailingaddress = inputData.getOrDefault("mailingAddressStreetAddress1", "").equals("") &&
+            inputData.getOrDefault("mailingAddressCity", "").equals("") &&
+            inputData.getOrDefault("mailingAddressState", "").equals("") &&
+            inputData.getOrDefault("mailingAddressZipCode", "").equals("");
 
-    if (noHomeAddress){
+    if (noHomeAddress && noMailingaddress) {
```

Comment:
Just trying to understand - is there a reason why this check needed to consider noMailingaddress as well? Is there an edge case?